pkgs/applications/graphics/gscreenshot/0001-Changing-paths-to-be-nix-compatible.patch

```diff
@@ -0,0 +1,36 @@
+From bbf31cb38dbe8e7ef4f02e055aefa3dc15418a99 Mon Sep 17 00:00:00 2001
+From: Richard Davis <davisrichard437@gmail.com>
+Date: Tue, 24 Jan 2023 19:11:25 -0500
+Subject: [PATCH] Changing paths to be nix-compatible
+
+---
+ dist/desktop/gscreenshot.desktop                    | 2 +-
+ src/gscreenshot/screenshooter/xdg_desktop_portal.py | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/dist/desktop/gscreenshot.desktop b/dist/desktop/gscreenshot.desktop
+index a5d2bcd..9d289e2 100644
+--- a/dist/desktop/gscreenshot.desktop
++++ b/dist/desktop/gscreenshot.desktop
+@@ -2,7 +2,7 @@
+ Type=Application
+ Name=gscreenshot
+ Comment=A simple screenshot utility
+-TryExec=/usr/bin/gscreenshot
++TryExec=gscreenshot
+ Exec=gscreenshot
+ Icon=gscreenshot
+ Categories=Graphics;
+diff --git a/src/gscreenshot/screenshooter/xdg_desktop_portal.py b/src/gscreenshot/screenshooter/xdg_desktop_portal.py
+index 78b4b71..914a285 100644
+--- a/src/gscreenshot/screenshooter/xdg_desktop_portal.py
++++ b/src/gscreenshot/screenshooter/xdg_desktop_portal.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/python3
++#!/usr/bin/env python3
```

Comment:
Isn't this shebang line re-written by `buildPythonApplication` anyway?

Reply:
I believe you are correct, that seems to be something I did in a broad swoop to get the `.desktop` file to work. I have just pushed the fix.